Automattic/wp-calypso
# Add Disconnect Jetpack Block

Part of tackling this issue 
https://github.com/Automattic/wp-calypso/issues/11861

To test
Go to dev http://calypso.localhost:3000/devdocs/blocks

And click on the Disconnect Jetpack Button 
You should see a Dialog that looks something like this 
Updated:

Free Plan
![image](https://cloud.githubusercontent.com/assets/115071/24020911/8125ff24-0a5b-11e7-8f5a-13f221a3c2d7.png)

Personal Plan
![image](https://cloud.githubusercontent.com/assets/115071/24020917/8a2e2074-0a5b-11e7-9b26-bb8e4b28fc1a.png)

Premium Plan
![image](https://cloud.githubusercontent.com/assets/115071/24020922/92e6d030-0a5b-11e7-8969-67d620b54aa2.png)

Professional Plan 
![image](https://cloud.githubusercontent.com/assets/115071/24020943/a8881d7c-0a5b-11e7-8c26-08cebd6bf61c.png)

And Broken 
![image](https://cloud.githubusercontent.com/assets/115071/24020966/c6809502-0a5b-11e7-8868-90b2a94407ea.png)


cc: @rickybanister, @keoshi
